android/build.gradle

```diff
@@ -117,8 +117,6 @@ android {
         exclude "**/libreact_render*.so"
     }
 
-    buildDir 'buildOutput_' + configStringPath
```

Comment:
Is this necessary really necessary ? The idea of this path configuration is to avoid cleaning project after enabling or disabling a build option...

Reply:
I think it's necessary for using New Architecture by react-native.
If I don't remove this `buildDir 'buildOutput_' + configStringPath` settings, I get this error when I gradle sync in Android Studio.
![Screenshot 2024-07-01 at 10 04 29 PM](https://github.com/TheWidlarzGroup/react-native-video/assets/36250002/30ed66b5-127d-4cf0-ad8b-1f9ee1ac17c3)
![Screenshot 2024-07-01 at 10 04 37 PM](https://github.com/TheWidlarzGroup/react-native-video/assets/36250002/740525ce-05c3-4b42-9450-64fb45989f32)

I think react-native's settings are the main issue, but I think it would be easier to follow their pre-setup path for codegen.

Reply:
So what is the main purpose of set up `buildDir 'buildOutput_' + configStringPath` in build.gradle?

Reply:
We use it in order to make "clean" build each time we change some android config (eg. extensions like IMA or exo-player)

Reply:
@KrzysztofMoch Understood. But it wouldn't be harmful by removing `buildDir 'buildOutput_' + configStringPath`
And also I think removing `buildDir 'buildOutput_' + configStringPath` is inevitable so far.

Reply:
So, I have a plan for resolving this issue.

If `buildDir` is the only reason that we cannot merge this PR, my idea is like this.

First, revert this commit https://github.com/TheWidlarzGroup/react-native-video/pull/3957/commits/62ec8d739b1f21906972eb935fc934e87294a8e6

I think we can test Android Fabric without removing `buildDir` in build.gradle.
And let's consider `buildDir` later when Fabric implementation is almost done for Android.

Reply:
I've pushed revert commit
[Revert "feat: 🔥 turn off buildDir setting in android/build.gradle for…](https://github.com/TheWidlarzGroup/react-native-video/pull/3957/commits/923b9d4592141bb11388f1cd99647147de5a3b67)

Can you have a check my opinions? @freeboub

Reply:
Sound good to me
btw now I understand why you want to remove it - I will do some research if we can "clean" build each time we change some android config in other way

Reply:
@KrzysztofMoch happy that you got my situation!
Your opinion would be another way to resolve this issue 👍

Reply:
ping @freeboub 
Can you check this thread please?
I think we don't need to delay about this PR.